### ISSUE TITLE
Script to find repeated jars is not getting the lowest version

### ISSUE THREAD

**Issue by bermud** [opengeospatial/teamengine-builder#6]
The script orders by string and not number 

If we have these jars it will suggest that the jar to be removed is geomatics-geotk-1.10.jar. It should be 1.9

```
-rw-r--r-- 1 tomcat7 tomcat    47966 Dec  3 16:27 geomatics-geotk-1.10.jar
-rw-r--r-- 1 tomcat7 tomcat    47931 Jul 30 09:44 geomatics-geotk-1.9.jar
```

**Comment by aashishlodha** [comment]
@keshav-nangare ,

In above files,
geomatics-geotk-1.9.jar and
geomatics-geotk-1.10.jar

In String comparison (character by character), the substring "geomatics-geotk-1.9" will come after "geomatics-geotk-1.1". The Comparator will not go for "0" (next character).

Solution:
I think, we should take out the version substring with regular expression, like the string between last '-' character and last '.' character.
For e.g, It should give substring "1.10" for string "geomatics-geotk-1.10.jar"

Then break that version string by '.' character into multiple strings, and run for loop on it, and compare the strings by parsing them into Integers, like in first iteration 1 > 1 (major versions)? then in second iteration 9 > 10 (minor versions)?

**Comment by bermud** [comment]
Fixed in commit 1fd49d6e9c38c8679f2b207a49baa6a155e94d34.
